Jessy/restaurants_challenge.py

```diff
@@ -0,0 +1,119 @@
+print("Challenge: Favourite Restaurants")
+
+print()
+
+print("Question 1")
+
+'''
+Below is a dictionary consisting of details of 1 restaurant fetched from Yelp. 
+
+Go through the dictionary and print out the following 3 pieces of information about the restaurant: 
+1. The latitude and longitude of Four Barrel Coffee 
+
+2. The complete address of Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code. 
+3. The website of Four Barrel Coffee
+'''
+
+
+restaurant = {
+    "name": "Four Barrel Coffee",
+    "url": "https://www.yelp.com/biz/four-barrel-coffee-san-francisco",
+    "latitude": 37.7670169511878,
+    "longitude": -122.42184275,
+    "city": "San Francisco",
+    "country": "US",
+    "address2": "",
+    "address3": "",
+    "state": "CA",
+    "address1": "375 Valencia St",
+    "zip_code": "94103",
+    "distance": 1604.23,
+    "transactions": ["pickup", "delivery"]
+}
+
+# print(restaurant)
+
+# TODO: Write code to print the latitude and longitude of Four Barrel Coffee.
+print(restaurant["latitude"])
+print(f'The Latitude and Longitude for Four Barrel Coffee is {restaurant["latitude"]} and {restaurant["longitude"]}')
+# TODO: Write code to print the complete address of the Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code.
+print(f'Them coordinates will take you to {restaurant["name"]} in {restaurant["address1"]} {restaurant["city"]} {restaurant["state"]} {restaurant["zip_code"]}')
+# TODO: Write code to print the URL of the website of Four Barrel Coffee.
+print(f"You may find more information at {restaurant['url']}")
+
+
+print()
+
+print("Question 2")
+
+# TODO: Choose 3 of your most favourite restaurants in NYC, and create a dictionary for each of them with the following key-value pairs:
+#         1. name : name of the resturant (string)
+#         2. address: address of the restaurant (string)
+#         3. favourite_dish: your favourite thing to order at the restaurant (string)
+restaurant_1  = {
+    "name": "Chaat Corner",
+    "address" : "320 3rd Street, San Francisco, 94103",
+    "favourite_dish" : "Chicken Tikka Masala" }
+
+restaurant_2  = {
+    "name": "The Fly Trap",
+    "address" : "606 Folsom Street, San Francisco, 94103",
+    "favourite_dish" : "Fried Brussel Sprouts" }
+
+
+restaurant_3  = {
+    "name": "Son & Garden",
+    "address" : "700 Polk Street, San Francisco, 94102",
+    "favourite_dish" : "Farmhouse Melt" }
+
+# TODO: Print each dictionary
+print(restaurant_1)
+"\n"
```

Comment:
the  `\n` needs to be inside a print function and inside quotes

print(  " \n " )